rpc_jobs/multi_node_aio.yml

```diff
@@ -3,16 +3,20 @@
     series:
       - mitaka:
           branch: mitaka-13.1
-          TEST_SET: "scenario defcore api heat_api smoke"
+          USER_VARS:
+            tempest_tests_sets: 'scenario defcore api heat_api smoke'
```

Comment:
I think test should be singular tempest_test**S**_sets also 3 other instances below.